src/test/java/org/jenkinsci/plugins/workflow/cps/SnippetizerTest.java

```diff
@@ -125,34 +154,43 @@
         BuildTriggerStep step = new BuildTriggerStep("downstream");
         assertRoundTrip(step, "build 'downstream'");
         step.setParameters(Arrays.asList(new StringParameterValue("branch", "default"), new BooleanParameterValue("correct", true)));
+        // when this test fails because of new core, see below "buildTriggerStepWith_2_2"
```

Comment:
:bug: due to `plugin-compat-tester` we do not wish to write tests which we _know_ will fail with a newer core. Make it pass on any sufficiently new core version, for example by checking `StringParameterValue.class.getAnnotation(Symbol.class) != null` and selecting the expected assertion accordingly.